NebulaHost/PacketProcessors/Factory/Entity/CreatePrebuildsRequestProcessor.cs

```diff
@@ -44,6 +44,10 @@ public void ProcessPacket(CreatePrebuildsRequest packet, NebulaConnection conn)
                     planet.physics = new PlanetPhysics(planet);
                     planet.physics.Init();
                 }
+                if (AccessTools.Field(typeof(CargoTraffic), "beltRenderingBatch").GetValue(planet.factory.cargoTraffic) == null)
```

Comment:
https://harmony.pardeike.net/api/HarmonyLib.AccessTools.html#HarmonyLib_AccessTools_FieldRefAccess__2___0_System_Reflection_FieldInfo_

Could be used instead. That way it wouldn't allocate every time.